signer/src/stacks/api.rs

```diff
@@ -902,11 +896,23 @@ impl StacksInteract for StacksClient {
     where
         T: AsTxPayload + Send + Sync,
     {
+        // Consensus serialize the transaction payload to bytes. This is the
+        // method that the stacks node uses to determine transaction size when
+        // verifying admittance into the mempool.
+        let mut payload_bytes = Vec::new();
+        payload
+            .tx_payload()
+            .consensus_serialize(&mut payload_bytes)
```

Comment:
I _think_ that this is the same [`consensus_serialize`](https://github.com/stacks-network/stacks-core/blob/40059a57cd27e740c5e9d91a833fb2c975b0bf0b/stackslib/src/core/mempool.rs#L2356) used to calculate the transaction size for admission into the mempool...

Reply:
You could also use `serialize_to_vec()`.

Reply:
Changed to use that